src/app/auth/pages/connection-page/connection-page.component.ts

```diff
@@ -14,20 +17,35 @@ export class ConnectionPageComponent implements OnInit {
   email?: string;
   pwd?: string;
 
-  constructor() {
+  constructor(
+    private userService: UserService,
+    private authService: AuthService,
+    private router: Router
+  ) {
   }
 
   ngOnInit(): void {
   }
 
   loginUser(form: NgForm) {
     // this.submitted = true;
-    const user: User = {
+    const currentUser: User = {
       email: form.value.email,
       password: form.value.pwd
     };
-    localStorage.setItem('email', user.email);
-    localStorage.setItem('pwd', user.password);
+    const user: User|null = this.userService.getByEmail(currentUser.email);
+
+    if (user != null) {
+      if (this.userService.verifPwd(currentUser, user)) {
+        this.authService.login(currentUser);
+        this.router.navigate(['/games']);
+      } else {
+        console.error("Mot de passe incorrecte");
+      }
+    } else {
+      console.error("L'utilisateur n'a pas été trouvé");
+    }
```

Comment:
je pense qu'on peut simplifier ça un peu : 
```suggestion
    if (this.userService.exists(userl) {
        this.authService.login(currentUser);
        this.router.navigate(['/games']);
      } else {
        console.error("email ou mot passe incorrect");
        // faire en sorte d'afficher un message d'erreur dans ton formulaire
    }
```